Revamp query expression related BBEs (#3901)
ballerina-platform/ballerina-distribution: examples/query-expressions/query_expressions.md

# Query expressions

> Query expression is similar to the `SQL` query syntax where you can construct a list, a mapping, a table, a stream or a sequence by iterating over an iterable value.

Reviewer: ```suggestion
Acquery expression is similar to the `SQL` query syntax where you can construct a list, a mapping, a table, a stream, or a sequence by iterating over an iterable value.
```

> The query expression consists of a sequence of clauses starting with a `from clause` and ends with a `select clause`.

Reviewer 2: ```suggestion
The query expression consists of a sequence of clauses starting with a `from` clause and ending with a `select` clause.
```

> The query expression consists of a sequence of clauses starting with a `from clause` and ends with a `select clause`.

Reviewer: ```suggestion
A query expression consists of a sequence of clauses starting with a `from` clause and ending with a `select` clause.
```

::: code query_expressions.bal :::

::: out query_expressions.out :::

## Related links

> - [Query expressions - Ballerina by example](https://ballerina.io/learn/by-example/query-expressions)

Reviewer: ```suggestion
- [Query expressions](https://ballerina.io/learn/by-example/query-expressions)
```
Fix all occurrences.
